nbdev/export2html.py

```diff
@@ -323,8 +334,8 @@ def get_metadata(cells):
                 cells.pop(i)
                 attrs = {k:v for k,v in _re_properties.findall(cell['source'])}
                 return {'keywords': 'fastai',
-                        'summary' : match.groups()[1],
-                        'title'   : match.groups()[0],
+                        'summary' : _md2html_links(match.groups()[1]),
+                        'title'   : _md2html_links(match.groups()[0]),
```

Comment:
Titles should be free of links so I'm not sure we need one here.

Reply:
Done!